toolchain/check/testdata/facet/require_import.carbon

```diff
@@ -36,6 +36,7 @@ fn F(A:! X, B:! Y) {}
 // CHECK:STDOUT:   %.Self: %type = symbolic_binding .Self [symbolic_self]
 // CHECK:STDOUT:   %X.type: type = facet_type <@X> [concrete]
 // CHECK:STDOUT:   %Self.861: %X.type = symbolic_binding Self, 0 [symbolic]
+// CHECK:STDOUT:   %Z.type: type = facet_type <@Z> [concrete]
```

Comment:
The PR description makes it sound like this is just a formatting change, but it looks like it's changing the underlying SemIR. I'm especially puzzled that stuff is getting added to the SemIR, when the underlying change is about removing something.